.gitlab-ci.yml

```diff
@@ -0,0 +1,30 @@
+#
```

Comment:
How does the GitLab repo stay up to date if we are pushing upstream to GitHub?

Reply:
It is a [pull mirror](https://docs.gitlab.com/ee/user/project/repository/repository_mirroring.html). We don't hack this, GitLab CI is supposed to be used like this (automatic mirror + CI)